linkerd/examples/acceptance-test.yaml

```diff
@@ -49,6 +49,11 @@ routers:
     httpUriInDst: true
   responseClassifier:
     kind: retryableIdempotent5XX
+  announcers:
+  - kind: io.l5d.announcer.Serversets
```

Comment:
I'd prefer just io.l5d.zk.serversets -- the fact that it's an announcer is self-evident